### ISSUE TITLE
Unexpected 403s

### ISSUE THREAD

**Issue by tontonsb** [tempestphp/tempest-docs#38]
Sorry, how do you make route params optional here? Or is it the nginx config that makes the 403 for whatever reason?

Anyway, I was looking at some page at the docs and wanted to go to the root of the docs, so I just deleted the trail of the URI and it turns out that https://tempestphp.com/docs/ doesn't work. And neither does https://tempestphp.com/docs. No luck with https://tempestphp.com/docs/framework/ either. There should be some reasonable redirects from those URIs or at least a 404 instead of the 403.

Btw I was expecting to discover the controller [here](https://github.com/search?q=repo%3Atempestphp%2Ftempest-docs%20docs%2Fframework&type=code). How comes the route is defined without the `docs/` prefix?

**Comment by brendt** [comment]
Yeah it's a problem with how nginx is configured. Thanks for pointing that out, I'll look into it!
